testapp/java/org/openyolo/testapp/AuthenticationMethodsInputView.java

```diff
@@ -103,7 +103,7 @@ private void initialize(Context context) {
         }
 
         if (mIdAndPasswordCheckBox.isChecked()) {
-            authenticationMethods.add(AuthenticationMethods.ID_AND_PASSWORD);
+            authenticationMethods.add(AuthenticationMethods.EMAIL);
         }
 
```

Comment:
For now, updated to just have email. Will open an issue to add phone and username.